src/app/core/components/header/header.component.scss

```diff
@@ -84,7 +84,7 @@ a:hover {
   background-size: 100% 2px;
 
   .page-header__icon {
-    background: rgba($color: var(--color-light), $alpha: 0.2);
+    background: rgba($color: #ffffff, $alpha: 0.2);
```

Comment:
Why did you change to #ffffff? ```var(--color-light)``` doesn't work?

Reply:
Yes, it doesn't work.